pom.xml

```diff
@@ -431,7 +431,7 @@
                 <plugins>
                     <plugin>
                         <groupId>org.codehaus.mojo</groupId>
-                        <artifactId>license-maven-plugin</artifactId>
+                        <artifactId>#license-maven-plugin</artifactId>
```

Comment:
@kshepherd : Is this a typo?  Not sure what the `#` sign is doing here

Reply:
Thanks, not sure how that got there either, I'll update shortly